app/views/layouts/application.html.erb

```diff
@@ -6,7 +6,7 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%# stylesheet_link_tag "application", "data-turbo-track": "reload" %>
```

Comment:
Why is this getting commented out?

Reply:
I commented it out because it was causing an error when running `rails server`, just forgot to put it back!